Simplified/AxisService/NYPLAxisService.swift

```diff
@@ -25,42 +25,26 @@ import Foundation
- 5. From the `container.xml file`, we extract the url for a `package.opf` file and download the
- package.opf file (`NYPLAxisService::downloadPackage`).
- 6. From the `package.opf` file (which is more of an xml really), we extract urls for all the required files for
- the book (e.g. chapter1.xtml, cover.html, fonts, images, styles etc.). We pass those urls in a for loop and tell
- the downloader to download all the files. (`NYPLAxisService::downloadAssetsFromPackage`)
- 7. When all the files are downloaded, we consider this as book download success and call the method associated
- with success on our NYPLBookDownloadBroadcasting delegate. (`replaceBook:withFileAtURL:forDownloadTask`)
- 8. If we experience a failure at any point during the download process, we stop the process, delete the
+ book.
+ 4. If the license download is successful, and the license is valid, we download two files, `Container.xml`
+ & `Encryption.xml` from Axis using the isbn key.
+ 5. From the `container.xml file`, we extract the url for a pacakge file and download the package and
+ download all the required book content mentioned in the package file.
+ 6. When all the files are downloaded, we consider this as book download success and call the method
+ associated with success  (`replaceBook:withFileAtURL:forDownloadTask`) on our
+ NYPLBookDownloadBroadcasting delegate.
+ 7. If we experience a failure at any point during the download process, we stop the process, delete the
  downloaded files, and call our delegate's `failDownloadWithAlert:` method.
  */
 @objc class NYPLAxisService: NSObject {
-  private weak var delegate: NYPLBookDownloadBroadcasting?
-  private let isbn: String
-  private let bookVaultId: String
-  private let dedicatedWriteURL: URL
-  private let fileURL: URL
-  private let baseURL: URL
+  private let axisItemDownloader: NYPLAxisItemDownloading
   private let book: NYPLBook
-  private let deviceInfoProviding: NYPLDeviceInfoProviding
-  private var packageEndpointproviding: NYPLAxisPackageEndpointProviding
-  private let packagePathProviding: NYPLAxisPackagePathPrefixProviding
-  private let axisKeysProviding: NYPLAxisKeysProviding
-  private let dispatchGroup: DispatchGroup
-  private let assetWriter: NYPLAssetWriting
-  private var downloader: NYPLAxisContentDownloading?
-  private var cypher: NYPLRSACryptographing?
-  private let globalBackgroundSyncronizeDataQueue = DispatchQueue(label: "NYPLAxis")
+  private let dedicatedWriteURL: URL
+  private let licenseService: NYPLAxisLicenseHandling
+  private let metadataDownloader: NYPLAxisMetadataContentHandling
+  private let packageDownloader: NYPLAxisPackageHandling
+  private weak var delegate: NYPLBookDownloadBroadcasting?
   
-  private var shouldContinue: Bool {
-    return globalBackgroundSyncronizeDataQueue.sync { [weak self] in
-      self?.downloader != nil
-    }
-  }
   
   // MARK: - Static Constants
   static let reason = "reason"
```

Comment:
since you're using this in crashlytics reports, can you change it to "axisDRMReason" or something like that? So that it's easier to search for it.